dev-ruby/ipaddr/ipaddr-1.2.5.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2022 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+USE_RUBY="ruby30 ruby31"
+RUBY_FAKEGEM_EXTRADOC="README.md"
+
```

Comment:
If you use a SRC_URI you will need to specific RUBY_FAKEGEM_GEMSPEC. It won't technically break in this specific situation because ipaddr has no dependencies, but we still try to use the actual gemspec in that case. It also avoids a warning in the ebuild.